src/cluster_slot_stats.c

```diff
@@ -4,14 +4,14 @@
  * SPDX-License-Identifier: BSD 3-Clause
  */
 
-#include "server.h"
-#include "cluster.h"
+#include "cluster_slot_stats.h"
 
 #define UNASSIGNED_SLOT 0
 
 typedef enum {
-    KEY_COUNT,
     INVALID,
+    KEY_COUNT,
+    CPU_USEC,
```

Comment:
On another Pr Hair mentioned having a "STAT_COUNT" metric be last so it could be used. INVALID can go after key count.